recipes/LibriSpeech/ASR/transformer/train_with_whisper.py

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env python3
+""" Take the encoder-decoder Whisper and the official tokenizer of whisper to fine-tune it with the NLL.
+
+This recipe can also be used as a zero-shot recipe by removing the training part.
+
+To run this recipe, do the following:
+> python train_with_whisper.py hparams/train_hf_whisper.yaml
+
+Authors
+ * Adel Moumen 2022
+ * Titouan Parcollet 2022
+"""
+
+import os
+import sys
+import torch
+import logging
+import speechbrain as sb
+from speechbrain.utils.distributed import run_on_main
+from speechbrain.utils.data_utils import undo_padding
+from hyperpyyaml import load_hyperpyyaml
+from pathlib import Path
+from transformers.models.whisper.tokenization_whisper import LANGUAGES
```

Comment:
this only works when you have git clone the transformers repo... I don't see any other way to fetch the appropriate language token. Any ideas @TParcollet ?

Reply:
actually, we could copy-paste the LANGUAGES dict directly into the python script...

Reply:
The PR looks great :D ! 
Just a little note: I experimented with whisper a little and found that the decoder can be fairly unpredictable ( because of the noisy data trained on), especially with background sounds. Even with the decoding strategies that are provided in the article, it can still be annoying when whispers persist for entire paragraphs that they have never stated on the wav. Perhaps fine-tuning the encoder and using it as CTC model only can help avoiding that